### PR TITLE
[icudata] Build data library using stubdata

### PR DESCRIPTION
**Pull request by nagineni**

### CODE REVIEW

File: icu4c/source/stubdata/Makefile.in [hunk 1]
```diff
@@ -0,0 +1,156 @@
+# Copyright (C) 2016 and later: Unicode, Inc. and others.
+# License & terms of use: http://www.unicode.org/copyright.html
+#******************************************************************************
+#
+#   Copyright (C) 1999-2015, International Business Machines
+#   Corporation and others.  All Rights Reserved.
+#
+#******************************************************************************
+## Makefile.in for ICU stubdata
```
Comment (anderco) [inline icu4c/source/stubdata/Makefile.in:9]: Do we actually use this file in the build?

File: icu4c/source/stubdata/stubdata.vcxproj [hunk 2]
```diff
@@ -0,0 +1,173 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (anderco) [inline icu4c/source/stubdata/stubdata.vcxproj:1]: Ditto. Is this file used?

**Review by anderco — APPROVED** [review]
I'm guessing you simply copied the stubdata folder from the icu repository? That would explain the unused files.

**Comment by nagineni** [comment]
> I'm guessing you simply copied the stubdata folder from the icu repository? That would explain the unused files.

Yes, copied the folder from upstream repo without any edits.

**Pull request merged**